Python/Salt n pepper noise removal/salt n pepper.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
```

Comment:
full code is not clear

Reply:
Which part is not clear for you @maulanaakbardj.